analysis/webservice/webmodel/NexusRequestObjectTornadoFree.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import re
+from datetime import datetime
+from decimal import Decimal
+
+from pytz import UTC
+from webservice.webmodel.RequestParameters import RequestParameters
+from webservice.webmodel.StatsComputeOptions import StatsComputeOptions
+
+
+class NexusRequestObjectTornadoFree(StatsComputeOptions):
```

Comment:
Will this be renamed once NexusRequestObject is phased out?

Reply:
Yes that was what I had in mind.